tsweb/log.go

```diff
@@ -45,6 +45,10 @@ type AccessLogRecord struct {
 	Bytes int `json:"bytes,omitempty"`
 	// Error encountered during request processing.
 	Err string `json:"err,omitempty"`
+	// The unique ID for this request. When a request fails due to an error, the
```

Comment:
I was going to say these docs aren't in the usual style (complete sentences starting with the name of the field) but then I noticed none of them are. At least fix this one and we can clean the others later?

Reply:
Yeah it seemed weird but I went with matching the surrounding comment format.

I've fixed this one.